templates/firstboot/mellanox_fw_update.yaml

```diff
@@ -49,14 +49,16 @@ resources:
           template: |
             #!/usr/bin/env python
 
-            from HTMLParser import HTMLParser
             import logging
             import os
             import re
             import shutil
             import tempfile
             import threading
-            import urllib2
+
+            from six.moves.html_parser import HTMLParser
```

Comment:
I would prefer that you import the module  html_parser and not the class HTMLParser.
I see that it was originally written like this, but it inconsistent other imports

Reply:
done